_pages/en/strategy-open-first.md

```diff
@@ -0,0 +1,445 @@
+- specifications and supporting material are freely available with limited restrictions; and
+- they are supported by the community and demonstrate independence or approved through due process by rough consensus among participants.
+
+Working across the ESDC, and joined by other GC departments, provincial Governments, companies and the public at large, we can build new, innovative digital solutions that support our common policies and activities, and work towards technological sovereignty.
+
+Open standards and OSS allows for incremental innovation, based on the sharing of knowledge and skills.
+Openness increases trust in public services.
+It helps to solve complex technological problems by getting others to contribute unexpected solutions.
+
+Being free to inspect and improve, OSS offers more options for increasing security.
+It allows for independent audits and code inspections, so the time and effort spent can be balanced according to need.
+This improves security, not just for us, but for everyone.
+
+This "Open First" strategy reinforces the ESDC's internal working culture that is already largely based on the principles of working in the open, open standards ans OSS.
+However, it will also help ESDC change some of its technological and information management processes.
+
+To build Canada together, we have to get ready to contribute, and be ready to accept contributions from citizens.
+
+The strategy is a practical instrument for achieving digital transformation in IITB, ESDC and the GC.
+At the same time, in a world of increasing uncertainty, complexity and change, the co-creation of technology increases our capacity to act independently to safeguard our interests, uphold Canadian values and way of life, and help shape our future.
+
+This strategy sets out how we encourage and leverage open principles and development practices to co-create, share and reuse across ESDC and the GC.
+By removing legal barriers, we can make it easier to contribute to OSS projects and to share our own solutions.
+The strategy brings ESDC closer to the OSS community, and makes us an agile contributor and an active participant.
+It encourages us to explore opportunities for dedicated support services for open solutions that we consider critical.
+We will make more and more use of open standards and OSS – in areas where it is practical, in the domain of IT, to areas where it makes sense and domains where it is strategic.
+
+## 2. Vision
+
+> ESDC leverages the transformative, innovative, and collaborative power of open standards and OSS, encouraging the sharing and reuse of solutions, knowledge and expertise, to deliver better Canadian services that enrich society and focus on lowering costs to that society.
+
+The title of this strategy, "Open First", points to a change in mindset whereby the acquisition and development of software solutions takes account of openness, sharing and reuse, security, privacy, legal considerations and accessibility.
```

Comment:
I think this would be a good place to provide a statement that ESDC's strategy was strongly influenced by the "Study on OSS governance at the European Commission".  That there is strong alignment between the EC's and ESDC's values and approach to open Source.